src/spaces/homspace.jl

```diff
@@ -158,6 +158,47 @@ function compose(W::HomSpace{S}, V::HomSpace{S}) where {S}
     return HomSpace(codomain(W), domain(V))
 end
 
+"""
+    insertunit(W::HomSpace, i::Int=ndims(W) + 1; conj=false, dual=false, preferdomain=false)
+
+Insert a trivial vector space, isomorphic to the underlying field, at position `i`.
+Whenever `i == numout(W)`, the ambiguity to determine whether this space is added in the domain or
+codomain is controlled by `preferdomain`.
```

Comment:
Is the motivation for "prefer..." that it actually only be added to the domain if `i>=numout(W)`? It is an atypical keyword name, but I cannot easily come up with something better.